### PR TITLE
Optimize the developer module

### PR DESCRIPTION
**Pull request by break60**

### CODE REVIEW

File: src/pages/development/index.md.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    return (
+      <div className="md2html development-page">
+        <Header
+          currentKey="development"
+          type="normal"
+          logo="/img/hlogo_colorful.svg"
+          language={language}
+          onLanguageChange={this.onLanguageChange}
+        />
+        <Bar img="/img/system/docs.png" text={dataSource.barText} />
```
Comment (chengshiwen) [inline src/pages/development/index.md.jsx:25]: Is it necessary to change to a new picture?

**Review by chengshiwen — COMMENTED** [review]
It's also necessary to search the old refer link globally, such as the keyword `docs/development`

**Comment by break60** [comment]
> It's also necessary to search the old refer link and repalce them by new link globally, such as the keyword `docs/development`

👌

**Comment by chengshiwen** [comment]
It's better to execute `npm run lint:fix`~

**Comment by break60** [comment]
> It's better to execute `npm run lint:fix`~

already fixed

**Comment by chengshiwen** [comment]
+1

**Pull request merged**